modules/core/core-api/src/main/java/com/enonic/xp/scheduler/ScheduleCalendar.java

```diff
@@ -10,6 +11,8 @@
 public interface ScheduleCalendar
     extends Serializable
 {
+    Optional<Instant> nextExecution( Instant instant );
```

Comment:
Do we need this method in every calendar, or does it make sense only in CronCalender?